src/view/configuration/configuration.jsx

```diff
@@ -233,7 +235,10 @@ const validationSchema = object()
   });
 
 const Configuration = () => {
-  const [selectedAccordionIndex, setSelectedAccordionIndex] = useState(0);
+  const [selectedAccordionIndex, setSelectedAccordionIndex] = useState();
+  const [isFirstExtensionViewRender, setIsFirstExtensionViewRender] = useState(
+    true
+  );
```

Comment:
This code is not related to tooltips. It helps make sure that the accordion is collapsed when a configuration is loaded that contains multiple instances (per UX design).